marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/search/getEntityInstance.sjs

```diff
@@ -160,14 +160,19 @@ function getInstanceFromTextDocument() {
   ]
 }
 
+function getInstanceWithInstanceInDifferentXpath() {
```

Comment:
@rahulvudutala  We still don't need this test :) - I was just raising a hypothetical. Though I'm curious - does es.instanceJsonFromDocument actually return the instance when "es:instance" is at any depth? 

I would normally say - there's no need to support "/*:envelope/*/es:instance". But if es.instanceJsonFromDocument can handle that, then I suppose it's okay to include.

Reply:
@rjrudin es.instanceJsonFromDocument is returning the instance when "es:instance" is at any depth
I tried on the document which has the path /es:envelope/es:merged/es:instance. The instance is returned (There is no  /es:envelope/es:instance in the document)

Reply:
Huh - well I guess that's good? Definitely surprising behavior, but I don't think there's a real downside to it.